resources/services/elasticbeanstalk/environments.go

```diff
@@ -450,6 +456,10 @@ func fetchElasticbeanstalkEnvironmentLinks(ctx context.Context, meta schema.Clie
 	return nil
 }
 
+func logInvalidParameterValueError(meta schema.ClientMeta, err error, p types.EnvironmentDescription) {
+	meta.Logger().Debug("Failed extracting configuration settings for environment. It might be terminated", "environment", p.EnvironmentName, "application", p.ApplicationName)
```

Comment:
configuruation settings or configuration options? maybe remove the fn and have two separate log lines in the respective call sites

Reply:
Ah, good catch. Let me fix it

Reply:
Done in https://github.com/cloudquery/cq-provider-aws/pull/751/commits/5cca3d4b154022366b35ac9d30f2d745dde702ff